themes/navy/layout/index.ejs

```diff
@@ -218,7 +218,10 @@
 				</div>
 				<div class="col-md-6 d-flex align-items-center justify-content-center">
 					<div class="link-arrow">
-						Assemble coming soon
+						<a href="https://assemble.fund/" class="link-arrow" target="_blank">
+						<svg width="18" height="17" viewBox="0 0 18 17" fill="none" xmlns="http://www.w3.org/2000/svg"> <path d="M9.25369 0.5C8.80652 0.5 8.35996 0.667031 8.01727 0.999494C7.33187 1.66442 7.33187 2.73446 8.01727 3.39939L11.5245 6.80275H2.24945C1.28015 6.80275 0.5 7.5605 0.5 8.50085C0.5 9.44119 1.28015 10.1981 2.24945 10.1981H11.5254L8.01727 13.6014C7.33187 14.2663 7.33187 15.3364 8.01727 16.0013C8.70266 16.6662 9.80564 16.6662 10.491 16.0013L16.986 9.70035C17.6713 9.03542 17.6713 7.96538 16.986 7.30045L10.491 0.999494C10.1483 0.66703 9.70087 0.5 9.25369 0.5Z" fill="#4360DF"/> </svg>
```

Comment:
Can you put the image in a separate file? They all sit under `themes/navy/source/img/*.svg`. It's easier to re-use them that way and makes for cleaner HTML.

Reply:
Good point, I've changed it to

<img src="/img/arrow-right.svg" alt="arrow-right">

@jakubgs

Reply:
Wait, let me change the line where I copied the svg code and others.

Reply:
Thanks!